.gitignore

```diff
@@ -6,6 +6,7 @@ binaries/*
 !binaries/README.md
 
 venv
+venv.
```

Comment:
`venv.`? Is that what gets created on Windows?

Reply:
yes it's what I get locally

Reply:
Could you add a comment ? Almost 100% sure that somebody will take it as a typo in the coming month :)